src/index.ts

```diff
@@ -0,0 +1,65 @@
+import * as cookies from 'js-cookie'
+
+function removeNaked (): string {
+  const domain = `${location.hostname}`
+  return domain.indexOf('www.') === 0 ? domain.substring(4) : domain
+}
+
+function setCookie (
+  domainParts: Array<string>,
+  name: string,
+  options: cookies.CookieAttributes,
+  data: string
```

Comment:
only `string` ?

Reply:
Yes. To set cookie,  value is required.

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/js-cookie/index.d.ts#L44

value maybe `string | object`

Reply:
oh. now, I checked js-cookie interface and will extend interface for fix this.

Reply:
I am waiting for a new PR :)